doc/mongo_extensions.py

```diff
@@ -75,7 +75,7 @@ def process_mongodoc_nodes(app, doctree, fromdocname):
                 anchor = name["ids"][0]
                 break
         for para in node.traverse(nodes.paragraph):
-            tag = str(para.traverse()[1])
+            tag = str(list(para.traverse())[1])
```

Comment:
Fixes:
```
[2020/08/10 23:25:41.924] /data/mci/54080f1ff7147a775a4b89567f2d3257/src/doc/mongo_extensions.py:78: FutureWarning:
--
[2020/08/10 23:25:41.924]    The iterable returned by Node.traverse()
[2020/08/10 23:25:41.924]    will become an iterator instead of a list in Docutils > 0.16.
[2020/08/10 23:25:41.924]   tag = str(para.traverse()[1])
```